package.json

```diff
@@ -1,57 +1,31 @@
 {
-  "name": "gnareact",
-  "version": "0.0.1",
-  "description": "The Gnar's React app starter kit",
-  "main": "dist/bundle.js",
-  "repository": "https://github.com/TheGnarCo/gnarlist.git",
-  "author": "Mike Stone <mike@gnar.dog>",
```

Comment:
Do we want to keep the author in here? Doesn't really matter but 🤷‍♂

Reply:
🤷‍♂ doesn't matter to me.